src/main.cc

```diff
@@ -819,6 +819,11 @@ int main(int argc, char* argv[])
         {
             if (not name.empty() and name[0_byte] == '+')
             {
+                if (name.begin()+1 == name.end() or *(name.begin()+1)  == ':')
```

Comment:
This check looks weird, are you sure you can't just check for the length of the `name` string and be done with it? Also apparently the `+:…` argument format is supported too?